client/app/components/packages/pas-form/project-geography.hbs

```diff
@@ -10,7 +10,7 @@
     </p>
 
     <div data-test-section="project-geography">
-      <ProjectGeography
+      <Packages::ProjectGeography
```

Comment:
Just leaving a note for the future. We may want to put this component in `<Packages::PasForm::ProjectGeography>`